examples/multigpu/graphbolt/node_classification.py

```diff
@@ -148,20 +148,10 @@ def create_dataloader(
 
 
 @torch.no_grad()
-def evaluate(rank, args, model, graph, features, itemset, num_classes, device):
+def evaluate(rank, model, dataloader, num_classes, device):
```

Comment:
Is this change related to peresistent worker? if not, move this to another PR.

Reply:
Okay